instrumentation-genai/opentelemetry-instrumentation-openai-v2/README.rst

```diff
@@ -17,6 +17,53 @@ Installation
 
     pip install opentelemetry-instrumentation-openai-v2
 
+Usage
+-----
+
+Instrumenting all clients
+*************************
+
+When using the instrumentor, all clients will automatically trace OpenAI chat completion operations
+and capture prompts and completions as events.
```

Comment:
Similar thought to @gyliu513 and also I don't think most people know what events are yet, specifically they are log not span. Maybe adding the word log helps?

```suggestion
When using the instrumentor, all clients will automatically trace OpenAI chat completion operations.
You can also optionally capture prompts and completions as log events, discussed later.
```